test/soca/CMakeLists.txt

```diff
@@ -0,0 +1,176 @@
+# Copy the bkg files
```

Comment:
My bad, for some reason I thought you placed your test  under gdas-utils.

Reply:
the tests are in gdas.cd/build/gdas/test
This is where Shastri's tests were.
I can put them elsewhere.